From d6b555e248acf3708e2b9215f9a03bb04e88c685 Mon Sep 17 00:00:00 2001
From: Jordan Justen <jordan.l.justen@intel.com>
Date: Thu, 16 Feb 2017 17:35:32 -0800
Subject: [PATCH 2/2] debian: Use file-magic instead of python-magic in
 setup.py

Signed-off-by: Jordan Justen <jordan.l.justen@intel.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index a3f6f90..1224d74 100755
--- a/setup.py
+++ b/setup.py
@@ -30,7 +30,7 @@ setup(name='alot',
         'urwid>=1.1.0',
         'urwidtrees>=1.0',
         'twisted>=10.2.0',
-        'python-magic',
+        'file-magic',
         'configobj>=4.7.0',
         'pygpgme>=0.2'],
       tests_require=[
-- 
2.11.0

